python/lsst/pipe/tasks/imageDifference.py

```diff
@@ -518,7 +526,10 @@ def run(self, sensorRef, templateIdList=None):
 
             if self.config.doMeasurement:
                 self.log.info("Running diaSource measurement")
-                self.measurement.run(diaSources, subtractedExposure)
+                if not self.config.doMeasurementNew:
+                    self.measurement.run(diaSources, subtractedExposure)
+                else:
+                    self.measurement.run(diaSources, subtractedExposure, exposure, templateExposure)
```

Comment:
I would like to do some further tests on real data (and/or Ian's simulated images) before officially pulling the plug on the old algorithm. This change is making those tests possible (DM-5412), and we could remove the old one immediately afterwards (new ticket to be filed).